app/presenters/two_factor_auth_code/max_attempts_reached_presenter.rb

```diff
@@ -32,7 +32,7 @@ def message
     end
 
     def next_steps
-      [please_try_again, read_about_two_factor_authentication]
```

Comment:
I think the `please_try_again` link here is the clock which is what is giving you this failure.